mkdocs.yml

```diff
@@ -209,6 +201,21 @@ nav:
     - Generating Knowledge Graphs with Structured Outputs: 'hub/knowledge_graph.md'
     - Extracting Relevant Clips from YouTube Videos: "hub/youtube_clips.md"
     - Building Knowledge Graphs with Structured Outputs: 'tutorials/5-knowledge-graphs.ipynb'
+  - Integrations:
+    - Structured outputs with Anyscale: 'integrations/anyscale.md'
```

Comment:
just make this "Anthropic" and remove the structured output refereces 
set the title in the md file